neb/src/callbacks.cc

```diff
@@ -853,23 +853,20 @@ int neb::callback_flapping_status(int callback_type, void* data) {
     flapping_status->event_time = flapping_data->timestamp.tv_sec;
     flapping_status->event_type = flapping_data->type;
     flapping_status->high_threshold = flapping_data->high_threshold;
-    if (!flapping_data->host_name)
+    if (!flapping_data->host_id)
       throw(exceptions::msg() << "unnamed host");
     if (flapping_data->service_description) {
```

Comment:
in case of a service_description not defined, let's say we have a flapping host, you can also use service_id = 0.

So:
if (flapping_data->service_id) {
should do the trick